lib/ddtrace/contrib/sinatra/tracer.rb

```diff
@@ -34,7 +34,7 @@ def route(verb, action, *)
           # Keep track of the route name when the app is instantiated for an
           # incoming request.
           condition do
-            @datadog_route = action
+            @datadog_route = "#{request.script_name}#{action}"
```

Comment:
This feature seems kind of an advanced use-case IMO. Could we prepend this conditionally, using a configuration param? I think for most users the resource name without the script name would still be a better default.

Reply:
I think that's reasonable. Any thoughts about what you'd call it?

Reply:
`script_name_prefix`?